Library/src/Systems/Sequence/SequenceSystem.cpp

```diff
@@ -103,7 +103,26 @@ void SequenceSystem::UpdateSequence(const String& SequenceKey,
 									const csp::common::Map<csp::common::String, csp::common::String>& MetaData,
 									SequenceResultCallback Callback)
 {
-	CreateSequence(SequenceKey, ReferenceType, ReferenceId, Items, MetaData, Callback);
+	if (!ValidateKey(SequenceKey))
+	{
+		CSP_LOG_FORMAT(csp::systems::LogLevel::Error,
```

Comment:
[Question] why not use CSP_LOG_ERROR_FORMAT?

Reply:
This was based on the format of the related code, so no particular reason so happy to update for readability.

Reply:
nah I was just curious. thanks!